### PR TITLE
list inactive users

### PR DESCRIPTION
**Pull request by ryanwoldatwork**
automate the report of users who haven't logged in in 90 days

### CODE REVIEW

File: spec/features/admin/users_spec.rb [hunk 1]
```diff
@@ -72,22 +84,37 @@
...
+      it "does not display Admin nor Last Sign in at" do
+        expect(page).not_to have_content("Admin")
+        expect(page).not_to have_content("Last signed in")
       end
     end
 
-    it "does not display Admin nor Last Sign in at" do
-      expect(page).not_to have_content("Admin")
-      expect(page).not_to have_content("Last signed in")
+    describe "/admin/users/inactive" do
```
Comment (ryanwoldatwork) [inline spec/features/admin/users_spec.rb:110]: this is the new code

File: spec/features/admin/users_spec.rb [hunk 2]
```diff
@@ -72,22 +84,37 @@
...
     end
 
-    it "permitted to see the page" do
-      expect(page).to have_content("Users")
-      within "table" do
-        expect(page).to have_content("Org mgr")
-        expect(page).to have_content("Email")
-        expect(page).to have_content("Organization Name")
-        expect(page).to have_link("Edit")
+    describe "admin" do
```
Comment (ryanwoldatwork) [inline spec/features/admin/users_spec.rb:89]: this is just indented into a describe block

**Review by aktfrikshun — APPROVED** [review]
I don't see any issues here.



We may want create a scheduled job that marks a user as inactive if they haven't logged in in 90 days ( in addition to other criteria we might add in the future for inactivating a user ).

**Comment by ryanwoldatwork** [comment]
I agree a job could improve this automation. Communication to the client is needed somewhere in that flow. TBD.

**Pull request merged**